src/blueapi/worker/reworker.py

```diff
@@ -114,6 +114,8 @@ def begin_task(self, task_id: str) -> None:
 
     def submit_task(self, task: Task) -> str:
         task_id: str = str(uuid.uuid4())
+        if isinstance(task, RunPlan):
+            self._ctx.plans[task.name].model.validate(task.params)
```

Comment:
From discussion: Call `_lookup_params` and cache result until plan starts rather than validating twice.